spec.md

```diff
@@ -23,42 +23,25 @@ A standard token interface is a basic building block for many applications on th
    	totalSupply : Nat; // token total supply
    	owner : Principal; // token owner
    	fee : Nat; // fee for update calls
-   	feeTo : Principal; // fee receiver
    }
    ```
 
-2. Status: token status info
-
-   ```js
-   type Status = {
-   	historySize : Nat; // total number of history transactions
-   	deployTime: Time.Time; // token canister deploy time in nanoseconds
-   	holderNumber : Nat; // token holder number
-   	cycles : Nat; // token canister cycles balance
-   }
-   ```
-
-3. TxReceipt: receipt for update calls, contains the transaction index or an error message
+2. TxReceipt: receipt for update calls, contains the transaction index or an error message
 
    ```js
    type TxReceipt = Result.Result<Nat, {
    	#InsufficientBalance;
    	#InsufficientAllowance;
    }>;
-   ```
-
-4. TxRecord: history transaction record
+   
+3. TxRecord: history transaction record
 
    ```js
    public type Operation = {
-   	#mint;
```

Comment:
I think burning, minting, init should also be in ERC-20+ standard. It is also not clear for me what an init transaction is.